core/src/test/kotlin/io/customer/sdk/core/di/EventBusTest.kt

```diff
@@ -0,0 +1,153 @@
+package io.customer.sdk.core.di
+
+import io.customer.commontest.BaseUnitTest
+import io.customer.sdk.communication.Event
+import io.customer.sdk.communication.EventBusImpl
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.test.TestScope
+import kotlinx.coroutines.test.runTest
+import org.amshove.kluent.internal.assertEquals
+import org.amshove.kluent.shouldBe
+import org.amshove.kluent.shouldBeEqualTo
+import org.junit.Test
```

Comment:
This whole module is utilizing `JUnit4`, `DiGraphTest` as well. So it would a scoop creep to move it to `JUnit5` and remove `BaseUnitTest`. 

But I agree it should be done in later PRs.

Reply:
Makes sense. We can probably put a ticket for it in Linear too.